app/assets/javascripts/scp-related-genes-ideogram.js

```diff
@@ -72,6 +85,31 @@ function showRelatedGenesIdeogram() { // eslint-disable-line
   ideoContainer.classList = 'show-related-genes-ideogram'
 }
 
+/**
+ * Get summary of related-genes ideogram that was just loaded or clicked
+ */
+function getRelatedGenesAnalytics(ideo) {
+  const props = Object.assign({}, ideo.relatedGenesAnalytics)
+
+  // Use DSP-conventional name
+  props['perfTime'] = props.time
+  delete props['time']
+
+  props['species'] = window.SCP.taxon
+  return Object.assign({}, props)
```

Comment:
why do we need another copy?  props is a fresh object

Reply:
Fixed in dbd7beb.